lib/frame.rb

```diff
@@ -0,0 +1,53 @@
+class Frame
+  attr_accessor :rolls, :bonus_rolls, :bonus_points
+
+  def initialize(io = Kernel)
+    @io = io
+    @rolls = []
+    @bonus_rolls = 0
+    @bonus_points = 0
+  end
+
+  def format_roll
```

Comment:
Bit unsure with this as in the Readme there's examples of like '3 3' and then the use of '/' with '5 / '? Again this is probably just something that needs to be explained to me